### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by hurlburtc** [PixelZerg/QuizletBot#8]
Because of an update, the bot says that it does not support any of the games.

**Comment by YonatanAhituv** [comment]
Yeah, it's quite old. TIME FOR SHAMELESS ADVERTISING. However, if you are still looking to be the very best, then I have created a match bot in python if you'd like to try it. You don't have to, seriously. But it's here:
https://github.com/AtomicCoding/Quizlet-Bot

**Comment by shrey150** [comment]
I fixed the problem with Micro Scatter being updated, PixelZerg updated it
